server.js

```diff
@@ -28,7 +28,7 @@ if (config.auth && config.auth.enabled) {
     dbClientManager.useStaticClient(true);
```

Comment:
Should remove the lines above this for checking for auth-direct-file.

Also, there's a block in the README to be removed that discusses file-based auth.